plugins/scroll-options/src/index.js

```diff
@@ -75,9 +75,14 @@ export class ScrollOptions {
       // Already enabled.
       return;
     }
-    const dragSurface = this.workspace_.getBlockDragSurface();
+
+    let element = this.workspace_.getBlockDragSurface().getSvgRoot();
```

Comment:
if the drag surface is not being used, I think `getBlockDragSurface` can return null. so before calling `getSvgRoot` we should check for null, and/or we should only do this check if `is3dSupported` is true.

So i think this is safest?

```js
let element;
const dragSurface = this.workspace_.getBlockDragSurface();
if (Blockly.utils.svgMath.is3dSupported() && dragSurface) {
  element = dragSurface.getSvgRoot();
} else {
  element = this.workspace_.svgGroup_;
}
```

Reply:
Great, thanks for reviewing! Yeah, I agree. It's also a little bit strange as of now, since `getBlockDragSurface` doesn't return null even when it is not supported.